src/test/java/plugins/AbstractAnalysisTest.java

```diff
@@ -171,6 +171,17 @@ public void should_send_mail_with_expanded_tokens() {
                 String.format("%s: %d-0-%d", title, getNumberOfWarnings(), getNumberOfWarnings()));
     }
 
+
+    /**
+     * Restart Jenkins if there is not required extension - see JENKINS-49026
+     */
+    protected void checkExtensionAreDeployed(String extensionName, String extensionPointName){
```

Comment:
Add another overloaded method that does not use the first parameter? Using `null` as valid parameter is not a good programming style.